common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
@robrap Still a bit confused. If the HTML text is wrapped with HTML in the python (when building up the HTML text), then are you NOT suppose to use HTML in the template? Would the template just look like below?

````
<span class="message">${html_msg}</span>